DigitalLearningSolutions.Data.Tests/DataServices/CourseDataServiceTests.cs

```diff
@@ -216,5 +216,34 @@ public void GetNumberOfActiveCoursesAtCentre_with_filtered_category_returns_expe
             // Then
             count.Should().Be(3);
         }
+
+        [Test]
+        public void GetCourseStatisticsAtCentreForCategoryID_should_return_course_statistics_correctly()
+        {
+            // When
+            const int centreId = 101;
+            const int categoryId = 0;

```

Comment:
Should these + expectedFirstCourse be under `// Given` section to keep the when/then clear